modules/reliability/jsx/columnFormatter.js

```diff
@@ -0,0 +1,26 @@
+function formatColumn(column, cell, rowData, rowHeaders) {
+// If a column if set as hidden, don't display it
+  if (loris.hiddenHeaders.indexOf(column) > -1) {
+    return null;
+  }
+
+  // Create the mapping between rowHeaders and rowData in a row object.
+  var row = {};
+  rowHeaders.forEach(function(header, index) {
+    row[header] = rowData[index];
+}, this);
+
+ 
+    if(column === 'PSCID' ) {
+        if (row["Manual Swap"]==="yes") {
+        return <td>{cell} <span className="error">manual</span> </td>;
+        }
+        if(row["Invalid"] === 'yes' ) {
```

Comment:
This still doesn't look formatted. Please run ESLint.
From you loris root run `node_modules/.bin/eslint --fix modules/reliability/`